plugins/XIVCombo/XIVCombo.json

```diff
@@ -3,5 +3,5 @@
 	"Name": "XIV Combo",
 	"Description": "This plugin condenses combos and mutually exclusive abilities onto a single button.",
 	"InternalName": "XIVCombo",
-	"AssemblyVersion": "1.0.0.0"
+	"AssemblyVersion": "1.0.0.1"
```

Comment:
Same thing here - thanks a lot! I'll update my samples asap!

Reply:
Should be good to go now, I think?